dna/src/lib.rs

```diff
@@ -373,14 +363,14 @@ pub mod tests {
     fn parse_with_defaults_zome() {
         let dna = Dna::new_from_json(
             r#"{
-                "zomes": [
-                    {}
-                ]
+                "zomes": {
+                    "0": {}
```

Comment:
i won't reject the PR for this, but it is confusing to have strings that look like indexes